### PR TITLE
README.md: added link to if-me.org with

### PR DESCRIPTION
**Pull request by andy5995**
prominent note to readers that installation
and configuration isn't required.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 # if me
 
-if me is a community for mental health experiences that encourages people to
+[_if me_](https://www.if-me.org/) is a community for mental health experiences that encourages people to
```
Comment (HashNotAdam) [inline README.md]: Please keep lines to 80 characters. In markdown, some exceptions are allowed for things like URLs or where a line break could cause difficulties with the flow of reading, however, most of the time you should introduce a new line.

Comment (andy5995) [reply README.md]: Done!

**Comment by andy5995** [comment]
Hmmm... now that I think of it, I should forget about using italics. Agreed?

**Comment by andy5995** [comment]
Thanks for reviewing this @julianguyen and @HashNotAdam

**Pull request merged**